app/build.gradle.kts

```diff
@@ -352,24 +352,44 @@ configurations.forEach { configuration ->
   configuration.exclude("com.google.protobuf", "protobuf-lite")
 }
 
+fun runCommand(command: String) {
+    val os = System.getProperty("os.name").toLowerCase()
+    if (os.contains("win")) {
```

Comment:
Could be replaced with `when`conditional logic, as you already have three cases, and as it is better for fallbacks.